etc/omero.properties

```diff
@@ -724,10 +724,12 @@ omero.ldap.referral=ignore
 # not just the username, email, etc, but also the
 # groups that the user is a member of.
 #
-# .. warning::
-#    Currently setting this to true the user will be
-#    removed from any groups to which they have been
-#    added outside of LDAP! Please use carefully.
+# .. note::
+#    Admin actions carried out in the clients may
+#    not survive this synchronization e.g. LDAP
+#    users removed from an LDAP group in the UI
+#    will be re-added to the group when logging in
+#    again after the synchronization.
```

Comment:
Note this text was added and reviewed in https://github.com/openmicroscopy/ome-documentation/pull/1240/files#diff-caa9d10f16401f5b6d59ede6dd6a2963R242